src/Migrate-From-Self-Hosted-to-EMS.md

```diff
@@ -51,8 +67,10 @@ Copy the following files and send to EMS Support:
 
 ## Stop Synapse
 
-**DO NOT START IT AGAIN AFTER THIS**
-Doing so can cause issues with federation and inconsistent data for your users
+**DO NOT START IT AGAIN AFTER THIS**  
+Doing so can cause issues with federation and inconsistent data for your users.
+
+While you wait for the database to export or files to transfer, you should edit or create the well-known files and DNS records to point to your EMS host. This can take a while to update so should be done as soon as possible in order to ensure your server wil function properly when the migration is complete.
```

Comment:
minor typo, change `wil` to `will`

Reply:
```suggestion
While you wait for the database to export or files to transfer, you should edit or create the well-known files and DNS records to point to your EMS host. This can take a while to update so should be done as soon as possible in order to ensure your server will function properly when the migration is complete.
```